src/main/app/documents/documentsClient.ts

```diff
@@ -9,22 +9,15 @@ export class DocumentsClient {
 
   constructor (public documentsUrl: string = `${claimStoreBaseUrl}/documents`) {
   }
-
-  getResponseCopy (claimExternalId: string): http.IncomingMessage {
+  getPDF (claimExternalId: string, documentTemplate: string): http.IncomingMessage {
```

Comment:
we shouldn't be passing strings around to choose which one we want,
I would suggest making this private and restoring the original functions which the only thing they do is pass the correct string
REQ

Reply:
@timja-kainos can you remove the block now that it has been changed please?